extensions/wikia/HTTPSSupport/tests/HTTPSSupportHooksTest.php

```diff
@@ -86,4 +86,78 @@ public function onBeforeResourceLoaderCSSMinifierDataProvider() {
 			]
 		];
 	}
+
+	public function testOnLinkerMakeExternalLink() {
+
+		$exampleString = '';
+		$exampleBool = false;
+		$exampleArray = [];
+
+		#Should not be upgraded to HTTPS
+
+		#external URL
+		$url = 'http://www.example.com';
+		$expectedResult = 'http://www.example.com';
+		HTTPSSupportHooks::onLinkerMakeExternalLink($url,$exampleString , $exampleBool, $exampleArray);
+		$this->assertEquals( $expectedResult, $url );
+
+		#multiple subdomain wikia.com
+		$url = 'http://ja.starwars.wikia.com';
```

Comment:
To make sure it does not break links, we should use url to articles, maybe even add query parameters.